portfolio/src/main/webapp/index.html

```diff
@@ -9,8 +9,39 @@
   <body>
     <div id="content">
       <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
-      <p>Click here to get a random message:</p>
+            <p> My name is Kevin Shao, and this is my portfolio! I live in Los Angeles, California, 
```

Comment:
nit: please remove the spaces after the tag. Same for below

Reply:
To clarify, it's the space between the tag and the beginning of the text "My name is ..."